### PR TITLE
Update Windows install instructions

### PR DESCRIPTION
**Pull request by neatorobito**
Add clarity by expanding all the necessary steps

### CODE REVIEW

File: _pages/install/on_windows.md [hunk 1]
```diff
@@ -2,15 +2,27 @@
 subtitle: On Windows (Preview)
 ---
 
-To easily install the Crystal Preview on Windows you can use [Scoop](https://scoop.sh/). 
+To easily install Crystal on Windows you can use the [Scoop](https://scoop.sh/) preview package. 
 **Be aware of the fact that Crystal on Windows is** [**not yet complete**](https://github.com/crystal-lang/crystal/issues/5430).
 
-Before you start the installation, you must ensure that the [required packages](https://github.com/neatorobito/scoop-crystal#requirements) are installed.
+## Requirements
+* Developer mode enabled in Settings
```
Comment (beta-ziliani) [inline _pages/install/on_windows.md:9]: Adding link to MS docs.

```suggestion
* Developer mode [enabled](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development) in Settings
```

File: _pages/install/on_windows.md [hunk 2]
```diff
@@ -2,15 +2,27 @@
...
+* [Scoop](https://scoop.sh/)
+* x64 Native Tools Command Prompt in Windows Terminal
```
Comment (straight-shoota) [inline _pages/install/on_windows.md:11]: This is a bit confusing. It's technically a prerequisite for `crystal`, but you can also install it via scoop.
I think we should either directly mention here that it's included in the installation instructions, or not name it here at all, only in the installation section.

Comment (beta-ziliani) [reply _pages/install/on_windows.md:11]: fair point, it should be consistent with the `git` requirement

Comment (neatorobito) [reply _pages/install/on_windows.md]: Agreed, I've pushed new updates that address this.

File: _pages/install/on_windows.md [hunk 3]
```diff
@@ -2,15 +2,30 @@
 subtitle: On Windows (Preview)
 ---
 
-To easily install the Crystal Preview on Windows you can use [Scoop](https://scoop.sh/). 
+To easily install Crystal on Windows you can use the [Scoop](https://scoop.sh/) preview package. 
 **Be aware of the fact that Crystal on Windows is** [**not yet complete**](https://github.com/crystal-lang/crystal/issues/5430).
 
-Before you start the installation, you must ensure that the [required packages](https://github.com/neatorobito/scoop-crystal#requirements) are installed.
+## Requirements
+* Developer mode [enabled](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development) in Settings
```
Comment (straight-shoota) [inline _pages/install/on_windows.md:9]:
```suggestion
* [Developer mode enabled](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development) in Windows Settings
```

File: _pages/install/on_windows.md [hunk 4]
```diff
@@ -2,15 +2,30 @@
...
+* [Scoop](https://scoop.sh/)
 
 ## Install
+Start by installing git and adding the crystal repository:
```
Comment (straight-shoota) [inline _pages/install/on_windows.md:13]:
```suggestion
Start by installing git (if you don't already have it available) and adding the crystal repository:
```

File: _pages/install/on_windows.md [hunk 5]
```diff
@@ -23,3 +38,6 @@ scoop update
 scoop update crystal
 ```
 
+## Troubleshooting
+Please see this [repo](https://github.com/neatorobito/scoop-crystal) for more information and support.
```
Comment (straight-shoota) [inline _pages/install/on_windows.md:42]:
```suggestion
Please see the [`neatorobito/scoop-crystal` repo](https://github.com/neatorobito/scoop-crystal) for more information and support.
```

**Review by straight-shoota — COMMENTED** [review]
I just have some minor suggestions for polishing up the wording.

**Review by straight-shoota — APPROVED** [review]

**Review by beta-ziliani — APPROVED** [review]

**Pull request merged**